src/Docker/Container.php

```diff
@@ -416,10 +397,9 @@ public function getRunCommand($containerId)
         $command .= " --volume " . escapeshellarg($this->dataDir . ":/data")
             . " --volume " . escapeshellarg($this->tmpDir . ":/tmp")
             . " --memory " . escapeshellarg($this->limits->getMemoryLimit($this->getImage()))
-            . " --memory-swap " . escapeshellarg($this->limits->getMemorySwapLimit($this->getImage()))
+            . ($this->getImage()->getSourceComponent()->hasNoSwap() ? " --memory-swap " . escapeshellarg($this->limits->getMemorySwapLimit($this->getImage())) : "")
```

Comment:
JFYI od Symfony 3.3 se dá volat takhle bez escapování: `$process = new Process(array('/path/command', '--flag', 'arg 1', 'etc.'));
`

Reply:
tak to je dost cool! zapomenout na `escapeshellarg` už se mi párkrát vymstilo.

Reply:
no akorat tady jsme jeste na 2.8

Reply:
jj, zkousel https://github.com/keboola/syrup/issues/73, ale je to temer neresitelny - je to deprecated, takze resenim je zbavit se toho a pak to bude v pohode